src/pages/SignInModal.js

```diff
@@ -0,0 +1,279 @@
+              <p>학생</p>
+              <SignInRadioInput
+                onChange={getInputValue}
+                type="radio"
+                name="user_type"
+                value="멘토"
+              />
+              <p>멘토</p>
+            </div>
+          </SignInForm>
+          <SignInForm>
+            <SignInTitle>이름 *</SignInTitle>
+            <SignInInput
+              onChange={getInputValue}
+              name="name"
+              placeholder="이름을 입력해주세요."
+              ref={nameInput}
+            />
+          </SignInForm>
+          <SignInForm>
+            <SignInTitle>기수 *</SignInTitle>
+            <SignInInput
+              disabled={isAbleInput()}
+              onChange={getInputValue}
+              name="batch"
+              type="number"
+              placeholder="숫자로만 입력해주세요. ex)21"
+              check="false"
+              ref={batchInput}
+            />
+          </SignInForm>
+          <SignInForm>
+            <SignInTitle>포지션 *</SignInTitle>
+            <PositionSelect name="position" onChange={getInputValue}>
+              <option value="Front-End">Front-End</option>
+              <option value="Back-End">Back-End</option>
+              <option value="Fullstack">Fullstack</option>
+              <option selected value="Undefined">
+                미정
+              </option>
+            </PositionSelect>
+          </SignInForm>
+          <SignInForm>
+            <SignInTitle>생일</SignInTitle>
+            <SignInInput
+              type="date"
+              onChange={getInputValue}
+              name="birthday"
+              placeholder="생일을 입력해주세요"
+            />
+          </SignInForm>
+          <SignInForm>
+            <SignInTitle>Blog</SignInTitle>
+            <SignInInput
+              onChange={getInputValue}
+              name="blog"
+              placeholder="블로그 URL을 입력해주세요"
+            />
+          </SignInForm>
+          <SignInForm>
+            <SignInTitle>GitHub</SignInTitle>
+            <SignInInput
+              onChange={getInputValue}
+              name="github"
+              placeholder="GitHub 을 입력해주세요"
+            />
+          </SignInForm>
+        </SignInContent>
+        <SubmitButton
+          useRef={submitButton}
+          onClick={postUserData}
+          disabled={!isAbleButton()}
+        >
+          입력 완료
+        </SubmitButton>
+      </SignInContainer>
+    </ModalContainer>
+  );
+};
+export default SignInModal;
+
+const ModalContainer = styled.section`
+  ${({ theme }) => theme.flexbox()};
+  height: 100%;
+  width: 100%;
+  padding: 20px;
+`;
+
+const MainLogo = styled.div`
+  font-size: 25px;
+  padding: 30px;
+  font-weight: 700;
+  color: ${({ theme }) => theme.colors.black};
+`;
+const SignInContainer = styled.section`
+  ${({ theme }) => theme.flexbox('column', 'center', 'stretch')};
+  margin-top: 20px;
+`;
+
+const SignIntext = styled.div`
+  color: ${({ theme }) => theme.colors.black};
+  margin: 10px 0px 20px 0px;
```

Comment:
순서 바꿔주세용민이
```suggestion
  margin: 10px 0 20px 
  color: ${({ theme }) => theme.colors.black};
```
추가로 0px의 경우 px 안적어주셔도 됩니다~!
그리고 마주 보는 방향의 경우, pixel값이 같으면 따로 안적어주셔도 돼요~!